issue.go

```diff
@@ -457,10 +457,12 @@ func (t Time) MarshalJSON() ([]byte, error) {
 // UnmarshalJSON will transform the JIRA date into a time.Time
 // during the transformation of the JIRA JSON response
 func (t *Date) UnmarshalJSON(b []byte) error {
-	// Ignore null, like in the main JSON package.
-	if string(b) == "null" {
+	s := strings.Trim(string(b), "\"")
+	// Ignore null and empty strings.
+	if s == "" || s == "null" {
 		return nil
 	}
+
 	ti, err := time.Parse("\"2006-01-02\"", string(b))
```

Comment:
This is a kinda weird handing to deal with strings. If we need to accommodate that we should parse them input bytes into a string and let the json package do it instead of trying to trim them off or include them in the format like this.

Reply:
that was a mistake, check now